lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced-tabbed.component.ts

```diff
@@ -0,0 +1,137 @@
+            ...this.startValue
+        };
+        this.submitValues();
+    }
+
+    setValue(value: { [key: string]: SearchDateRangeAdvanced }) {
+        this.value = value;
+    }
+
+    submitValues() {
+        this.context.queryFragments[this.id] = this.combinedQuery;
+        this.displayValue$.next(this.combinedDisplayValue);
+        if (this.id && this.context) {
+            this.context.update();
+        }
+    }
+    onDateRangedValueChanged(value: Partial<SearchDateRangeAdvanced>, field: string) {
+        this.value[field] = value;
+        this.updateQuery(value, field);
+        this.updateDisplayValue(value, field);
+    }
+
+    private generateQuery(value: Partial<SearchDateRangeAdvanced>, field: string): string {
+        let query = '';
+        if (value.dateRangeType === DateRangeType.IN_LAST) {
+            query = `${field}:[NOW/DAY-${value.inLastValue}${value.inLastValueType} TO NOW/DAY+1DAY]`;
+        } else if (value.dateRangeType === DateRangeType.BETWEEN) {
+            query = `${field}:['${formatISO(startOfDay(value.betweenStartDate))}' TO '${formatISO(endOfDay(value.betweenEndDate))}']`;
+        }
+        return query;
+    }
+
+    private generateDisplayValue(value: Partial<SearchDateRangeAdvanced>): string {
+        let displayValue = '';
+        if (value.dateRangeType === DateRangeType.IN_LAST && value.inLastValue) {
+            displayValue = this.translateService.instant(`SEARCH.DATE_RANGE_ADVANCED.IN_LAST_DISPLAY_LABELS.${value.inLastValueType}`, {
+                value: value.inLastValue
+            });
+        } else if (value.dateRangeType === DateRangeType.BETWEEN && value.betweenStartDate && value.betweenEndDate) {
+            displayValue = `${format(startOfDay(value.betweenStartDate), this.settings.dateFormat)} - ${format(endOfDay(value.betweenEndDate), this.settings.dateFormat)}`;
+        }
+        return displayValue;
+    }
+
+    private updateQuery(value: Partial<SearchDateRangeAdvanced>, field: string) {
+        this.combinedQuery = '';
+        this.queryMapByField.set(field, this.generateQuery(value, field));
+        this.queryMapByField.forEach((query: string) => {
+            if (query) {
+                this.combinedQuery = this.combinedQuery ? `${this.combinedQuery} AND ${query}` : `${query}`;
+            }
+        });
+    }
+
+    private updateDisplayValue(value: Partial<SearchDateRangeAdvanced>, field: string) {
+        this.combinedDisplayValue = '';
+        this.displayValueMapByField.set(field, this.generateDisplayValue(value));
+        this.displayValueMapByField.forEach((displayValue: string, field: string) => {
+            if (displayValue) {
+                const displayLabelForField = `${this.translateService.instant(this.settings.displayedLabelsByField[field]).toUpperCase()}: ${displayValue}`;
+                this.combinedDisplayValue = this.combinedDisplayValue ? `${this.combinedDisplayValue} ${displayLabelForField}` : `${displayLabelForField}`
```

Comment:
missing semicolon

Reply:
These has been resolved in [this](https://github.com/Alfresco/alfresco-ng2-components/pull/8714/commits/bdaa0e52fd3a31ba7c3802c6cdc6a27a733d6057) commit